src/node_file.cc

```diff
@@ -3130,22 +3130,34 @@ static void CpSyncCheckPaths(const FunctionCallbackInfo<Value>& args) {
   ToNamespacedPath(env, &src);
   THROW_IF_INSUFFICIENT_PERMISSIONS(
       env, permission::PermissionScope::kFileSystemRead, src.ToStringView());
+
+#ifdef _WIN32
```

Comment:
It seems fine to always pass the reinterpreted chars, no matter it's Windows or not.